### ISSUE TITLE
self-reject instructions seems wrong or outtodate

### ISSUE THREAD

**Issue by maxandersen** [chocolatey/chocolatey.org#814]
### What You Are Seeing?
docs about self-reject package at https://chocolatey.org/docs/chocolatey-faqs#how-do-i-self-reject-a-package does not seem to reflect reality. screenshots and explanation does not work.

### What is Expected?
explanation on how to self-reject has screenshot of current version of website or somehow give info on how to self-reject.

**Comment by maxandersen** [comment]
btw. main issue to report this is that I'm trying to update https://chocolatey.org/packages/jbang/0.6.0.5 with a new version but upload on site gives error and choco push says one can't update package that are in review.

so i'm stuck and trying to self-rejct to see if that helps - but that’s stuck too due to not being able to find info.

**Comment by AdmiringWorm** [comment]
@maxandersen 

When a package haven't failed the verifier or validator, then you can't self-reject the package.
To get the package version rejected, you will need to use the `Contact Site Admins` link, and explain that you want the package version rejected, and why you want it rejected.

> new version but upload on site gives error and choco push says one can't update package that are in review.

The error message you get from `choco push` tells you exactly why you can't push a "new" version.
When there are an already submitted package version, which have yet to be approved/exempted or rejected, then you can not push a new version (only update the same version already submitted).

I do agree that the docs for self-rejecting packages should be updated though, but such issues should be filed over at: https://github.com/chocolatey/choco-wiki/issues

**Comment by ferventcoder** [comment]
Second sentence in the main paragraph: `If however you just want to reject a working package because it is older, we don't support that.`

Is there a way we can call that out a bit better?

**Comment by ferventcoder** [comment]
The image to me is pretty self-explanatory (only visible if the package has failed automated review). Again, if you have any info on how to better call that out, I agree with @AdmiringWorm and we could move this issue over there.

**Comment by ferventcoder** [comment]
@maxandersen Did you have any follow up on what I mentioned? We'd love to make the docs better and if you have input on that, that would be fantastic.

**Comment by maxandersen** [comment]
well, the screenshot is not what is shown on the website. the check box button is not there.

**Comment by ferventcoder** [comment]
@maxandersen the check box button is only there if the package is failing as per the documentation. 

Should we put something there and gray it out and then direct you to the docs to understand or put something there so when you hover you can see you are not in a state that can actually self-reject?

**Comment by ferventcoder** [comment]
Basically, that wouldn't be a change to the docs, but to the UI to show that you can't do it.

**Comment by maxandersen** [comment]
Yeah, so my issue was that I could see my initial submission had errors/issues so was stuck in the queue with no way to remove or update before the first approval.

I read self-reject instructions as "if you made a mistake you can mark it as self-rejected"  - you are saying that is actually not possible and only viewable if some part of testing failed ?

**Comment by AdmiringWorm** [comment]
> Yeah, so my issue was that I could see my initial submission had errors/issues so was stuck in the queue with no way to remove or update before the first approval.

When there is a mistake in the package submitted, and it has yet to be approved/exempted or rejected, typically you would push the same version, so it replaces the one already pushed.

> I read self-reject instructions as "if you made a mistake you can mark it as self-rejected" - you are saying that is actually not possible and only viewable if some part of testing failed ?

Correct, you can only self-reject a package when at least 1 part of the automated checks have failed (either the verifier, or the validator).
IMO, that is clear with the mention in the screenshot on the documentation site (*although the screenshot does not match the current layout anymore*):

> Only visible if the package has failed automated review (validation/verification)

with an arrow pointing to the text `Reject Package?...`

> Should we put something there and gray it out and then direct you to the docs to understand or put something there so when you hover you can see you are not in a state that can actually self-reject?

I think that could be the best option, and it would also make it more transparent to the user (*perhaps*) without having to dig through the documentation.
